libs/components/storybook/.storybook/preview.ts

```diff
@@ -1,10 +1,9 @@
-import {
-  previewWrapperDecorators,
-  previewWrapperGlobalTypes,
-  previewWrapperParameters,
-} from '@skyux/storybook';
 import { moduleMetadata } from '@storybook/angular';
 
+import { previewWrapperDecorators } from '../src/lib/preview-wrapper/preview-wrapper-decorators';
```

Comment:
This is another linter found place where we were referencing the module in the module itself.